src/shared/Footer.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+interface IFooterProps {
+    borderThickness: string;
+}
+const Footer = styled.footer<IFooterProps>`
+    z-index: 11;
+    position: fixed;
```

Comment:
Following up on my last comment, i think it'd be best if we just had it positioned normally instead of fixed on the bottom.

Reply:
When removing `fixed`, the footer sticks to the bottom of the last component: 

![image](https://user-images.githubusercontent.com/17463268/51567216-a9739d00-1e64-11e9-80f4-d631b745b7a0.png)

How can we fix this?